boa3_test/tests/boa_test.py

```diff
@@ -178,25 +188,53 @@ def get_contract_path(self, *args: str) -> str:
             raise FileNotFoundError(path)
         return path
 
+    def get_deploy_file_paths_without_compiling(self, contract_path: str) -> Tuple[str, str]:
+        file_path_without_ext, _ = os.path.splitext(contract_path)
+        if USE_UNIQUE_NAME:
```

Comment:
Why use `USE_UNIQUE_NAME` instead of `self._use_custom_name`?

Reply:
`self._use_custom_name` was included in BoaTest to pass its value to the tests using TestEngine without having to import the `USE_UNIQUE_NAME` constant across all the test files, it'll be removed when we remove TestEngine as well